micropy/config/config.py

```diff
@@ -139,9 +113,12 @@ def get(self, key: str, default: Any = None) -> Any:
             Any: Value at key given
 
         """
-        key_path = key.split('.')
-        value = iterutils.get_path(self.config, key_path, default=default)
-        return value
+        try:
+            value = dpath.util.get(self.config, self.root(key))
+        except KeyError:
+            value = default
+        finally:
+            return value
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [return statement in finally block may swallow exception](https://app.codacy.com/app/BradenM/micropy-cli/pullRequest?prid=4594550)